src/uast.cc

```diff
@@ -33,6 +29,7 @@ struct UastIterator {
   TreeOrder order;
   std::deque<void *> pending;
   std::set<void *> visited;
+  void* (*nodeTransform)(void*);
```

Comment:
How about
```
using TransformFunc = (void*)(*)(void*);

TransformFunc nodeTransform;
```

and using that type in e.g. `UastIteratorNewWithTransformer`

Reply:
Thought about it, but it's pretty simple and only used in a function, but it's certainly more explicit so I'll change it.